src/handlers/handle-guild-message-creation.ts

```diff
@@ -0,0 +1,27 @@
+import type { Message } from 'discord.js';
+import { MessageType } from 'discord.js';
+
+export const handleGuildMessageCreation = async (message: Message) => {
+  if (message.author.bot) {
+    return;
+  }
+
+  if (message.type !== MessageType.Default) {
+    return;
+  }
+
+  const urls = message.content.match(/https?:\/\/\S+/g);
```

Comment:
hi, can you please make a check if there is some twitter links in the message ? Because the bot will send a message on every message.